kubernetes/loculus/values.yaml

```diff
@@ -1485,6 +1485,7 @@ defaultOrganisms:
         taxon_id: 3052518
         nextclade_dataset_server: https://raw.githubusercontent.com/nextstrain/nextclade_data/cornelius-cchfv/data_output
         nextclade_dataset_name: nextstrain/cchfv/linked
+        grouping_ground_truth: https://anna-parker.github.io/influenza-a-groupings/results/cchf_groups.json
```

Comment:
I added this so that the preview can test the grouping - currently we actually don't group the reference sequences so this should be removed before merging at the moment